admin/app/model/AdminLifecycle.scala

```diff
@@ -93,6 +93,10 @@ trait AdminLifecycle extends GlobalSettings with Logging {
       VideoEncodingsJob.run()
     }
 
+    Jobs.scheduleEveryNMinutes("AssetMetricsCache", 60 * 6) {
```

Comment:
Fetching the metrics from Cloudwatch is done via a schedule job in order to avoid unnecessary requests that cost money.
I set the cron job to run every 6 hours because it is unlikely we would like to track those stats at a smaller granularity but I am happy to discuss about this value